chill-scala/src/test/scala/com/twitter/chill/KryoSpec.scala

```diff
@@ -299,7 +299,7 @@ class KryoSpec extends WordSpec with Matchers with BaseProperties {
       toList(rt(qr)) should equal(qrlist)
     }
     "Ranges should be fixed size" in {
-      val MAX_RANGE_SIZE = 188 // what seems to be needed.
+      val MAX_RANGE_SIZE = 447 // what seems to be needed.
```

Comment:
:frowning:

We could make this constant a function of Scala version if you'd like to keep using the old value for the old build matrix entries.